frontend/src/api/apollo-client.ts

```diff
@@ -22,7 +22,7 @@ const headerLink = setContext((_, { headers }) => {
 });
 
 const httpLink = new HttpLink({
-  uri: JSON.parse(process.env.NEXT_PUBLIC_DEV) ? `http://localhost:8000/graphql/` : `https://api.test-pledge4future.heigit.org/graphql/`
```

Comment:
This is not required anymore I believe? Correct me if I'm wrong though

Reply:
No we still need this during development to test against a local backend.